### DISCUSSION TITLE
GPU Hang Exploration: Splitgate | clusterduck - pixelcluster's GPU blog

### DISCUSSION THREAD

**Discussion by giscus** [pixelcluster/pixelcluster.github.io#4] [Comments]
# GPU Hang Exploration: Splitgate | clusterduck - pixelcluster's GPU blog

https://pixelcluster.github.io/Hang-Exploration-Splitgate/

**Comment by Dantali0n** [comment]
You and your mates at Valve are doing awesome work on Mesa really, thanks so much for these amazing contributions to the FOSS / Linux gaming communities. Thanks to Valve for being such an awesome employer and thanks to everyone else contributing to Mesa, AMDGPU and Linux.
